<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunerDialog</class>
 <widget class="QDialog" name="TunerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuner</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FrequencyEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PresetEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Preset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PresetPlusButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Preset up</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PresetMinusButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Preset down</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassAButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassBButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassEButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassCButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassDButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassFButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassGButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset1Button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset2Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset3Button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset4Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset5Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset6Button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset7Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset8Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Preset9Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Preset
Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AMButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AM</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="FMButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FM</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="FrequencyMinusButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="FrequencyPlusButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ChoosePresetButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>161</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose a named preset</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DisplayButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DISP</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PTYSearchButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PTY
search</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NoiseCutButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOISE
CUT</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save the frequency in the preset or save the preset name</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="avrpioremote.qrc">
     <normaloff>:/new/prefix1/images/save.png</normaloff>:/new/prefix1/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="EditFrequencyButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edit the frequency and save it in the current preset.</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="avrpioremote.qrc">
     <normaloff>:/new/prefix1/images/pen.png</normaloff>:/new/prefix1/images/pen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="RenamePresetButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edit the current preset name</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="avrpioremote.qrc">
     <normaloff>:/new/prefix1/images/pen.png</normaloff>:/new/prefix1/images/pen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="CancelButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cancel the edit</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="avrpioremote.qrc">
     <normaloff>:/new/prefix1/images/cancel.png</normaloff>:/new/prefix1/images/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply the frequency</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="avrpioremote.qrc">
     <normaloff>:/new/prefix1/images/ok.png</normaloff>:/new/prefix1/images/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="DisplayEditBig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="avrpioremote.qrc"/>
 </resources>
 <connections/>
</ui>
